cmd/traffic/cmd/manager/internal/mutator/agent_injector.go

```diff
@@ -197,7 +197,7 @@ func hidePorts(pod *corev1.Pod, cn *corev1.Container, portName string, patches [
 	hidePort := func(path string) {
 		patches = append(patches, patchOperation{
 			Op:    "replace",
-			Path:  fmt.Sprintf("%s/%s/name", containerPath, path),
+			Path:  fmt.Sprintf("%s/%s", containerPath, path),
```

Comment:
While this change fixes the probe ports, it will no longer work on the container port. For the container it is indeed the value of the port's "name" key that should be replaced. I think that can be fixed by appending the `/name` suffix at 207.

Reply:
Good catch! Fixed according to your suggestion